src/app/add-movie/add-movie.component.html

```diff
@@ -0,0 +1,51 @@
+<p>add-movie works!</p>
+
+<div class="pt-3 pb-2 mb-3 border-bottom">
+    <h1 class="h2">Add Movie</h1>
+    <p class="custom-text">Add a <strong>movie</strong></p>
+  </div>
+  
+  <!-- Movie form -->
+  <form [formGroup]="moviesForm" (ngSubmit)="submitMovieData()" novalidate>
+    <div fxLayout="row">
+      <div class="col-lg-5 col-md-12 col-sm-12">
+        <div class="row">
+  
+          <div class="col-md-12 mb-3">
+            <label>Title of movie</label>
```

Comment:
Tu peux rajouter un attribut `for` à ton label pour le rattacher à ton input, rajoute aussi un `id` sur l'input qui sera identique au `for` du label